src/cli/info.rs

```diff
@@ -58,14 +60,18 @@ impl Display for EnvironmentInfo {
         let bold = console::Style::new().bold();
         writeln!(
             f,
-            "{}",
-            console::style(self.name.clone()).bold().blue().underlined()
+            "{:>WIDTH$}: {}",
+            bold.apply_to("Environment"),
+            self.name.fancy_display().bold()
         )?;
         writeln!(
             f,
             "{:>WIDTH$}: {}",
             bold.apply_to("Features"),
-            self.features.join(", ")
+            self.features
+                .iter()
+                .map(|feature| feature.fancy_display())
+                .join(", ")
```

Comment:
Better to use [`format`](https://docs.rs/itertools/latest/itertools/trait.Itertools.html#method.format) instead of `join`. `join` will allocate a string wheras `format` will not require additional heap allocation.